ipe_process_orchestrator/orchestrator.py

```diff
@@ -104,7 +107,9 @@ def start_composing_process(self):
         """
         This is the place where all the IPE process flow will be orchestrated.
         """
-        self._clean_up_ipe_dataframe()
+        self.filter_course_list_to_run_and_cleanup()
+        if self.filter_df_course_ids.empty:
```

Comment:
add a log message here when the filtered list is empty?

Reply:
I did not add the log statement here but during the filtering step. Below sample of it. I just generalized statement when you have and don't have courses to run. Look [here](https://github.com/tl-its-umich-edu/IPE-competencies-updater/pull/19/files#diff-76281cfa507df5613d7e2e1df0658848ad5188ee6495760bdb32aef8087f595fR40)
```
ipe_process_orchestrator.orchestrator - INFO - 0 courses to run the IPE process: []
```